common/src/main/java/com/mapbox/vision/common/view/BaseTeaserActivity.kt

```diff
@@ -84,8 +84,8 @@ abstract class BaseTeaserActivity : BaseVisionActivity() {
 
         override fun onFrameSignClassificationsUpdated(frameSignClassifications: FrameSignClassifications) {
             if (appMode == AppMode.Classification) {
+                tracker.update(UiSign.getUiSigns(frameSignClassifications))
```

Comment:
Well now there is possibility for threading issues, wdyt?

Reply:
Guess you're right - that could be an uncertainty. Fixed